dataland-qa-service/build.gradle.kts

```diff
@@ -37,6 +38,29 @@ dependencies {
     implementation(libs.log4j.to.slf4j)
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("org.springframework.amqp:spring-rabbit-test")
+
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation(libs.springdoc.openapi.ui)
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    runtimeOnly("org.postgresql:postgresql")
+    runtimeOnly("com.h2database:h2")
+    implementation(project(":dataland-keycloak-adapter"))
+    implementation("org.springframework.boot:spring-boot-starter-validation")
+    implementation("org.json:json:20230227")
```

Comment:
Question. Should the version be part of the build.gradle files or moved to the settings.gradle file?

Reply:
I think the settings.gradle would be preferrable because we have our version management mor centralized and uniform over all projects if the library is used in multiple projects.